tdt/dsp_server.py

```diff
@@ -10,7 +10,10 @@
 import os
 
 # Required so that sphinx can build this on readthedocs.org
-from . import actxobjects
+try:
+    from . import actxobjects
+except Exception:
+    pass

```

Comment:
@buranconsult I added this and nested some imports so that on linux I can at least do:
```
python -c "import tdt; print(tdt.__version__)
```
for testing purposes